gointelowl/job.go

```diff
@@ -0,0 +1,242 @@
+package gointelowl
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+type UserDetails struct {
+	Username string `json:"username"`
+}
+
+type Report struct {
+	Name                 string                 `json:"name"`
+	Status               string                 `json:"status"`
+	Report               map[string]interface{} `json:"report"`
+	Errors               []string               `json:"errors"`
+	ProcessTime          float64                `json:"process_time"`
+	StartTime            time.Time              `json:"start_time"`
+	EndTime              time.Time              `json:"end_time"`
+	RuntimeConfiguration map[string]interface{} `json:"runtime_configuration"`
+	Type                 string                 `json:"type"`
+}
+
+type Job struct {
+	ID                       int                    `json:"id"`
+	User                     UserDetails            `json:"user"`
+	Tags                     []Tag                  `json:"tags"`
+	ProcessTime              float64                `json:"process_time"`
+	AnalyzerReports          []Report               `json:"analyzer_reports,omitempty"`
+	ConnectorReports         []Report               `json:"connector_reports,omitempty"`
```

Comment:
Can you confirm if these fields are empty or omitted in job response?

Reply:
These are in the job response. However, they are not included in jobList.